Export fails on OSX

As I noted in the issue about switching to chrome as the default, a long time ago, the default settings do not work on OSX because they use the wrong pixel ratio:

![iris](https://user-images.githubusercontent.com/1078448/74006088-80b72c00-492f-11ea-9e5f-4715ddc80b74.png)

> In general, I need test output 
```
[08:39:36] Finished 'test:size' after 693 ms
[08:39:36] Starting 'test:start:headless'...
[08:39:36] Failed 'test:start:headless' after 4 ms
[08:39:36] Starting 'test:start:server'...
listening on 127.0.0.1:5777
[08:39:38] Finished 'test:start:server' after 1.73 s
[08:39:38] Starting 'defaults:generate'...
[08:39:39] generate_defaults.py: Wrote models_defaults.json with 179 model classes
[08:39:39] generate_defaults.py: Wrote widgets_defaults.json with 23 model classes
[08:39:39] Finished 'defaults:generate' after 1.27 s
[08:39:39] Starting 'test:compile'...
[08:39:43] Finished 'test:compile' after 3.97 s
[08:39:43] Starting 'test:unit:bundle'...
[08:39:48] Finished 'test:unit:bundle' after 5.07 s
[08:39:48] Starting 'test:integration:bundle'...
[08:39:49] Finished 'test:integration:bundle' after 44 ms
[08:39:49] failed: task 'test' failed
[08:39:49]     at _run (/Users/bryan/work/bokeh/bokehjs/make/task.ts:134:26)
[08:39:49]     at process._tickCallback (internal/process/next_tick.js:68:7)
```

That's it.

Also FYI there is some flakiness in using chromium an CI, just restarted a test with this
> E       selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally
E         (chrome not reachable)
E         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)

--- a/bokeh/io/export.py
+++ b/bokeh/io/export.py
@@ -198,10 +198,13 @@ def get_screenshot_as_png(obj: Union[LayoutDOM, Document], *, driver: Optional[W
         web_driver.maximize_window()
         web_driver.get("file:///" + tmp.path)
         wait_until_render_complete(web_driver, timeout)
-        [width, height] = _maximize_viewport(web_driver)
+        [width, height, dpr] = _maximize_viewport(web_driver)
         png = web_driver.get_screenshot_as_png()
 
-    return Image.open(io.BytesIO(png)).convert("RGBA").crop((0, 0, width, height))
+    return (Image.open(io.BytesIO(png))
+                 .convert("RGBA")
+                 .crop((0, 0, width*dpr, height*dpr))
+                 .resize((width, height)))
 
 def get_svgs(obj: Union[LayoutDOM, Document], *, driver: Optional[WebDriver] = None, timeout: int = 5,
         resources: Resources = INLINE, width: Optional[int] = None, height: Optional[int] = None) -> List[str]:
@@ -316,19 +319,19 @@ def _log_console(driver: WebDriver) -> None:
         for message in messages:
             log.warning(message)
 
-def _maximize_viewport(web_driver: WebDriver) -> Tuple[int, int]:
+def _maximize_viewport(web_driver: WebDriver) -> Tuple[int, int, int]:
     calculate_viewport_size = """\
         const root = document.getElementsByClassName("bk-root")[0]
         const {width, height} = root.children[0].getBoundingClientRect()
-        return [width, height]
+        return [width, height, window.devicePixelRatio]
     """
-    viewport_size: Tuple[int, int] = web_driver.execute_script(calculate_viewport_size)
+    viewport_size: Tuple[int, int, int] = web_driver.execute_script(calculate_viewport_size)
     calculate_window_size = """\
-        const [width, height] = arguments
+        const [width, height, dpr] = arguments
         return [
             // XXX: outer{Width,Height} can be 0 in headless mode under certain window managers
-            Math.max(0, window.outerWidth - window.innerWidth) + width,
-            Math.max(0, window.outerHeight - window.innerHeight) + height,
+            Math.max(0, window.outerWidth - window.innerWidth) + width*dpr,
+            Math.max(0, window.outerHeight - window.innerHeight) + height*dpr,
         ]
     """
     [width, height] = web_driver.execute_script(calculate_window_size, *viewport_size)

--- a/bokeh/io/webdriver.py
+++ b/bokeh/io/webdriver.py
@@ -63,6 +63,7 @@ def create_chromium_webdriver() -> WebDriver:
     options = webdriver.chrome.options.Options()
     options.add_argument("--headless")
     options.add_argument("--hide-scrollbars")
+    options.add_argument("--force-device-scale-factor=1")
     return webdriver.Chrome(options=options)
 
 #-----------------------------------------------------------------------------

